packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -79,6 +79,10 @@ export class DefaultPageBlockComponent
       overflow-y: auto;
       height: 100%;
     }
+
+    .dg.ac {
```

Comment:
Can we move this style into `DebugMenu`?

Reply:
The css scope in DebugMenu can't affect on dat-gui element🫠

Reply:
There seems can be a way to work

Reply:
You can inject the style globally from JavaScript side in `DebugMenu`, which can be also better than hard-coded here.